docs/Store.md

```diff
@@ -15,17 +15,19 @@ way of doing things.
 
 The store is queried each time one of the following properties is set on the instance:
 
-* store: references a `dstore/api/Store` implementation
+* source: references a `dstore/api/Store` implementation or an array
 * query: a query object to be passed to the store `filter()` function
```

Comment:
Should say "or an array **or decor/ObservableArray**", right?

Also, the documentation on the `query` property needs to be updated because it's talking about the dstore `filter()` method, but that doesn't make sense if the user is specifying an array rather than a dstore.